CODEOWNERS

```diff
@@ -2,13 +2,13 @@
 * @astarche @mgumble
```

Comment:
According to the GitHub docs about CodeOwners I think this works as a catch all. So if there was a file that didn't match the below patterns, Mose would be added. But if all the files in a PR matched the below patterns previously Mose wouldn't be as a code owner on the PR.

PR where all below patterns match and Mose was added manually (not marked code owner): #488 
vs.
This PR where the CODEOWNERS file doesn't match any of the specific patterns below and thus goes to this one (lowest matching pattern gets precedence) and Mose was added as an owner.

These changes should make Mose an owner on all PRs.